ai/gpu docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nvidia/cuda:12.2.0-devel-ubuntu20.04
```

Comment:
Is this dockerfile only for training? We should use different dockerfile for training and prediction.